WooCommerce/Classes/ViewRelated/Dashboard/Settings/CardReadersV2/CardReaderSettingsUnknownViewModel.swift

```diff
@@ -7,6 +7,7 @@ enum CardReaderSettingsUnknownViewModelDiscoveryState {
     case failed(Error)
     case foundReader
     case connectingToReader
+    case cancellingSearch
```

Comment:
I'm thinking this could be confusing in the future, since you won't go into this state when you actually tap cancel. How about `restartingSearch` instead?

Reply:
Done!